engine/access/apiproxy/access_api_proxy.go

```diff
@@ -256,14 +266,14 @@ func (h *FlowAccessAPIRouter) GetEventsForBlockIDs(context context.Context, req
 }
 
 func (h *FlowAccessAPIRouter) GetNetworkParameters(context context.Context, req *access.GetNetworkParametersRequest) (*access.GetNetworkParametersResponse, error) {
-	res, err := h.AccessAPIServer.GetNetworkParameters(context, req)
-	h.log("observer", "GetNetworkParameters", err)
+	res, err := h.Upstream.GetNetworkParameters(context, req)
+	h.log("upstream", "GetNetworkParameters", err)
```

Comment:
Certain methods were switched from local to upstream since the `protocol.API` interface does not implement them.

Reply:
https://github.com/onflow/flow-go/pull/2962/files#diff-3505449f31926a810a7226adb4a630986055d7196c265e57ac8633cc703ccb53R16-R51